sentry/src/main/java/io/sentry/MeasurementUnit.java

```diff
@@ -0,0 +1,132 @@
+package io.sentry;
+
+import java.util.Locale;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * The unit of measurement of a metric value.
+ *
+ * <p>Units augment metric values by giving them a magnitude and semantics. There are certain types
+ * of units that are subdivided in their precision, such as the {@link MeasurementUnit.Duration} for
+ * time measurements.
+ *
+ * <p>When using the units to custom measurements, Sentry will apply formatting to display
+ * measurement values in the UI.
+ *
+ * @see <a href="https://getsentry.github.io/relay/relay_metrics/enum.MetricUnit.html#">Develop
+ *     Docs</a>
+ */
+public interface MeasurementUnit {
+
+  /** Untyped value. */
+  String NONE = "none";
```

Comment:
Should users be able to pass `NONE` to `setMeasurement` explicitly? They can't if it's a String since it expects a `MeasurementUnit` to be passed.

Reply:
Nope, they should use the setMeasurement overload without a unit for that purpose. We could mark NONE internal actually.

Reply:
Sounds good